chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/ProductVersionManager.java

```diff
@@ -135,6 +137,9 @@ public ProductVersionDTO create(ProductVersionDTO dto)
     @Transactional(readOnly = false)
     @PreAuthorize("@permissions.hasAccess(T(gov.healthit.chpl.permissions.Permissions).PRODUCT_VERSION, "
             + "T(gov.healthit.chpl.permissions.domains.ProductVersionDomainPermissions).UPDATE, #dto)")
+    @CacheEvict(value = {
+            CacheNames.COLLECTIONS_LISTINGS
```

Comment:
I don't see a cache evict for `CacheNames.COLLECTIONS_LISTINGS` for Version create though it's there for Developer and Product create

Reply:
Added